test/index.js

```diff
@@ -296,16 +296,15 @@ describe('.dest()', function() {
 
   it('outputs files where the base ends in a separator', function(done) {
```

Comment:
I think we need to invert this test because the normalization was inverted in 2.0